Tests/Optiq.DatePicker.Tests/DatePicker/DatePickerUnitTests.cs

```diff
@@ -0,0 +1,210 @@
+﻿using Xunit;
+using Optiq.DatePicker;
+
+namespace Optiq.DatePicker.Tests;
+
+public class DatePickerUnitTests
+{
+    [Fact]
+    public void TestMinimumDate()
+    {
+        Optiq.DatePicker.DatePicker picker = new Optiq.DatePicker.DatePicker();
+
+        picker.MinimumDate = new DateTime(1950, 1, 1);
+
+        Assert.Equal(new DateTime(1950, 1, 1), picker.MinimumDate);
+
+        
+        Assert.Throws<ArgumentException>(() => picker.MinimumDate = new DateTime(2200, 1, 1));
```

Comment:
this is actually 2 test cases ("fact"s) in one test

Reply:
yeah.. please create separate tests here